package.json

```diff
@@ -23,41 +23,45 @@
   "main": "build/chronoshift.js",
   "typings": "build/index.d.ts",
   "files": [
-    "build/",
-    "typings/"
+    "build/"
   ],
   "scripts": {
-    "prepare": "tdi -q",
-    "test": "./travis-test",
-    "compile": "./compile"
+    "test": "jest",
+    "compile": "webpack -p",
+    "watch": "webpack -w"
   },
-  "config": {
-    "blanket": {
-      "pattern": "chronoshift/build/"
-    }
+  "jest": {
+    "preset": "ts-jest",
+    "testEnvironment": "jsdom",
+    "moduleNameMapper": {
+      "\\.s?css$": "identity-obj-proxy"
+    },
+    "testMatch": [
+      "**/?(*.)+(spec).ts?(x)"
+    ],
+    "setupFilesAfterEnv": [
+      "jest-expect-message"
+    ]
   },
   "dependencies": {
-    "immutable-class": "~0.7.0",
-    "moment-timezone": "~0.5.21",
-    "tslib": "~1.9.3"
+    "immutable-class": "~0.7.7",
+    "moment-timezone": "~0.5.25",
+    "tslib": "~1.10.0"
   },
   "devDependencies": {
-    "@types/mocha": "5.2.5",
-    "babel-core": "6.26.3",
-    "babel-preset-es2015": "6.24.1",
-    "blanket": "1.2.3",
-    "browserify": "16.2.2",
-    "chai": "4.1.2",
-    "gulp": "3.9.1",
-    "gulp-mocha": "6.0.0",
-    "gulp-typescript": "5.0.0-alpha.3",
+    "@types/jest": "^24.0.15",
+    "@types/jest-expect-message": "^1.0.0",
+    "@types/moment-timezone": "^0.5.12",
+    "@types/node": "^12.0.8",
```

Comment:
could you add awesome-code style for linting also please

Reply:
👍